statsmodels/tsa/tests/test_exponential_smoothing.py

```diff
@@ -719,6 +719,20 @@ def test_results_vs_statespace(statespace_comparison):
         statespace_results.fittedvalues.values
     )
 
+    # compare diagnostics
+    assert_almost_equal(
```

Comment:
You need to wrap the assert almost_equal with

```

with pytest.warns(FutureWarning):
   assert_almost_equal(...
```

since this warning is transformed into an error to make sure we aren't leaking any into a test run.